src/rules/expect-expect.ts

```diff
@@ -1,10 +1,35 @@
 import { Rule } from 'eslint';
 import * as ESTree from 'estree';
-import { isExpectCall, isTest } from '../utils/ast';
+import { isExpectCall, isIdentifier, isTest } from '../utils/ast';
+import { Settings } from '../utils/types';
+
+type Options = {
+  additionalAssertFunctionNames?: readonly string[];
+};
+
+function isAssertionCall(
+  node: ESTree.CallExpression,
+  additionalAssertFunctionNames: readonly string[]
+) {
+  return (
+    isExpectCall(node) ||
+    additionalAssertFunctionNames.find((name) =>
+      isIdentifier(node.callee, name)
+    )
+  );
+}
+
+function getAdditionalAssertFunctionNames(context: Rule.RuleContext): readonly string[] {
+  const globalSettings = (context.settings as Settings).playwright?.additionalAssertFunctionNames ?? [] as readonly string[]
+  const ruleSettings = (context.options[0] as Options | undefined)?.additionalAssertFunctionNames ?? [] as readonly string[]
+
+  return [...globalSettings, ...ruleSettings]
+}
```

Comment:
Maybe move this to a utils file for reuse

Reply:
It's not necessarily useful for other rules, considering its dependency on the rule's own options (`context.options[0]` in this case). Likewise, the issues with re-using this logic elsewhere that I outlined in my previous PR comment. Happy to move it if you'd like, I'm just not sure that it's a coherent signal to contributors in the future about how applicable this particular function is to rules aside from this one. Are there particular use-cases you have in mind that I could add to a doc comment on the function if it is moved?

Reply:
There are other rules that would benefit from the global settings, so yes let's move it out.